### ISSUE TITLE
Bulk Edit Work Order - Can't edit the fields with tick mark 

### ISSUE THREAD

**Issue by casakthiveln** [frappe/erpnext#31382]
### Information about bug

While editing the work order in bulk, the fields which requires tick marks are not working but other fields apart from tick marks are working fine. The following video will show elaboratly
![Work order bulk edit](https://user-images.githubusercontent.com/63579207/174046778-86c9884f-8428-4b66-9824-6d4d1e2b0c23.gif)
 

### Module

manufacturing

### Version

ERPNext: v13.34.0 (version-13)
Frappe Framework: v13.33.0 (version-13)
Production

**Comment by dj12djdjs** [comment]
This is a framework bug. Just tested, It's broken for all doctypes.

**Comment by treasuryesc** [comment]
> This is a framework bug. Just tested, It's broken for all doctypes.

Yes, we also have seen that it applies to all fields with type = Check
